### DISCUSSION TITLE
How to using gesture handle

### DISCUSSION THREAD

**Discussion by ozan1338** [react-navigation/react-navigation#10267] [Q&A]
Hello Guys I have a question how can enable gesture in react native
![enablegesture](https://user-images.githubusercontent.com/93523224/149627910-eb5bd60e-dce1-4162-9be4-5578156daea8.PNG)
i was search for about an hour and everyone just said put gestureEnable to true and boom they working but mine was not working and what i want was just screen of login and register got enable of using gesture but in todolist not
Sorry im newbie and sorry for my bad english👏

**Comment by raajnadar** [comment]
Read the docs https://reactnavigation.org/docs/stack-navigator#installation
